msal/src/main/java/com/microsoft/identity/client/controllers/MSALTokenCommand.java

```diff
@@ -0,0 +1,75 @@
+package com.microsoft.identity.client.controllers;
+
+import android.content.Context;
+import android.content.Intent;
+
+import com.microsoft.identity.client.AuthenticationCallback;
+import com.microsoft.identity.client.AuthenticationResult;
+
+public class MSALTokenCommand implements MSALTokenOperation {
```

Comment:
License is needed at the top of file, also could you please run the auto formatter on this file.

Reply:
added to all files.